### DISCUSSION TITLE
Inconsistency with observed velocity and written output

### DISCUSSION THREAD

**Discussion by rbenamran** [CliMA/Oceananigans.jl#3104] [Computational science!]
Hi, 

I've been having issues with the post-simulation analysis and noticed the velocities I am reading from the JLD2 output file are not what I expect. 

I specifically wrote a test simulation below with a Gaussian initial condition that propagates at a surface gravity wave speed of about 10 m/s for the given parameters (this is observed clearly through the animation). However, the velocities I read from the opened JLD2 file are never beyond two meters per second. 

Here is a plot of the velocity field as taken from the output JLD2 file. The structure is correct, but the magnitude is scaled down, because the total sim time is 3 second and the Gaussian starts at the center and reaches the edge, thus the velocity must be closer to 10m/s
![plot_3](https://github.com/CliMA/Oceananigans.jl/assets/86665862/ebde7668-52bc-439f-ab12-f6e2a048986b)

Here is the Gaussian sim code just in case.
```
using Oceananigans
using Oceananigans.Models: ShallowWaterModel

Nx, Ny = 64, 64   #grid nodes
Lx, Ly, Lz = 60, 60, 10  #grid size

f = 1   #coriolis
g = 9.81 #gravitational_acceleration       

c = sqrt(g * Lz)

grid = RectilinearGrid(size = (Nx, Ny),
                            x=(0, Lx), y=(0, Ly),
                            topology = (Periodic, Periodic, Flat)
)

model = ShallowWaterModel(
    grid = grid,
    gravitational_acceleration = g,
    coriolis = nothing,
    # formulation = ConservativeFormulation(),
    # momentum_advection = VectorInvariant()
)

A = 10   #amplitude
x₀ = Lx/2
y₀ = Ly/2

#Initial condition test function
h₀(x, y, z) = Lz + A * exp( - ((x - x₀)^2 + (y - y₀)^2)/4 )

set!(model, h=h₀)

uh, vh, h = model.solution
u = uh/h
v = vh/h
w = sqrt(u^2 + v^2)

simulation = Simulation(model, Δt=0.001, stop_iteration=3000)

using Oceananigans.OutputWriters: JLD2OutputWriter, IterationInterval

simulation.output_writers[:fields] =
    JLD2OutputWriter(
        model,
        (; u, v, w),
        filename = "test2.jld2",
        schedule=IterationInterval(1)
)

run!(simulation)
```

Thanks in advance for any insight on this!

**Comment by glwagner** [comment]
The magnitude of the velocity field and the _speed of propagation_ are different. If your signal is a shallow water wave, then it _propagates_ at the group speed,

$$ c = \sqrt{g H} \approx 9.9$$

where $g = 9.81$ is gravitational acceleration and $H = 10$ is the depth that you prescribed. So your result is expected.

You can test this by changing $A$ and observing that the time it takes for the wave to reach the edge of the domain remains the same.
